conda.recipe/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "dev" %}
+
+package:
+  name: netcdf-c
+  version: {{ version }}
+
+source:
+  path: ../
+
+build:
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - python  # Here only to activate the `vc`s.
+    - msinttypes
+    - curl
+    - zlib 1.2.*
+    - hdf4
+    - hdf5 1.8.17|1.8.17.*
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py>=35]
+  run:
+    - curl
+    - zlib 1.2.*
+    - hdf4
+    - hdf5 1.8.17|1.8.17.*
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py>=35]
+
+test:
+  commands:
+    - ncdump -h http://geoport-dev.whoi.edu/thredds/dodsC/estofs/atlantic
+    # FIXME: the next two are failing at the moment.
```

Comment:
Thes URLS are either not reliable (geoport-dev.whoi.edu) or cycle off (the GFS URL with a specific date).
This could be a good one as:
1. It's at Unidata
2. It doesn't reference a model resolution that may disappear someday (e.g. 0.5=>0.25) and it doesn't specify a particular date
``
ncdump -h http://thredds.ucar.edu/thredds/dodsC/grib/NCEP/WW3/Global/Best
```